x-pack/test/functional/services/ml/stack_management_jobs.ts

```diff
@@ -274,23 +285,116 @@ export function MachineLearningStackManagementJobsProvider(
     },
 
     async importJobs() {
-      await retry.tryForTime(5000, async () => {
-        await testSubjects.click('mlJobMgmtImportImportButton', 1000);
+      await testSubjects.click('mlJobMgmtImportImportButton', 1000);
+      await testSubjects.missingOrFail('mlJobMgmtImportJobsFlyout', { timeout: 60 * 1000 });
+    },
+
+    async assertReadErrorCalloutExists() {
+      await testSubjects.existOrFail('~mlJobMgmtImportJobsFileReadErrorCallout');
+    },
+
+    async selectExportJobType(jobType: JobType) {
+      if (jobType === 'anomaly-detector') {
+        await testSubjects.click('mlJobMgmtExportJobsADTab');
+        await testSubjects.existOrFail('mlJobMgmtExportJobsADJobList');
+      } else {
+        await testSubjects.click('mlJobMgmtExportJobsDFATab');
+        await testSubjects.existOrFail('mlJobMgmtExportJobsDFAJobList');
+      }
+    },
+
+    async selectExportJobSelectAll() {
+      await testSubjects.click('mlJobMgmtExportJobsSelectAllButton');
```

Comment:
Would be good to make sure that the click came through successfully. My first idea was to verify that the `select all` changed to `deselect all`, but this doesn't happen: it's still `select all`, while actually it deselects all items when clicking it again. Not sure if this is something we want to change. Another option could be to validate that all check boxes in the list are ticked after the click.

Reply:
Changed the label so it changes to `Deselect all`
Added a validation that all checkboxes are checked after clicking.

https://github.com/elastic/kibana/pull/110578/commits/62e901212a167e8db5384455296567e8b13963a8